schemas/log.yml

```diff
@@ -21,3 +21,14 @@
       description: >
         Offset of the beginning of the log event.
       example: 12
+    - name: message
+      type: keyword
+      phase: 1
+      example: "Sep 19 08:26:10 localhost My log"
+      description: >
+        This is the raw log message and contains the full log message before processing.
+
+        In contrast to the `message` field which can contain an extracted part
+        of the log message, this field contains the raw log message and should
+        not be processed. It can have already some modifications like encoding
```

Comment:
This sounds like a contractiction: 'raw log message' and 'not be processed' vs. 'can have already some modifications'

Reply:
Agree. Any suggestion for a better description instead of raw we could use here?